src/expressions/inExpression.ts

```diff
@@ -83,11 +84,47 @@ export class InExpression extends ChainableUnaryExpression {
   }
 
   protected _getSQLChainableUnaryHelper(
-    _dialect: SQLDialect,
-    _operandSQL: string,
+    dialect: SQLDialect,
+    operandSQL: string,
     _expressionSQL: string,
   ): string {
-    throw new Error(`can not convert action to SQL ${this}`);
+    const expression = this.expression;
+    const expressionType = expression.type;
+    switch (expressionType) {
+      case 'STRING_RANGE':
```

Comment:
I am not familiar with the string_range type but would it be possible to add a test to demonstrate its usage?

Reply:
Added. Much easier than it seemed when I originally looked at it 😆